app/backplane/radio_module/include/c_receiver_module.h

```diff
@@ -37,9 +39,9 @@ class CReceiverModule : public CProjectConfiguration {
     void SetupCallbacks() override;
 
 private:
-    static constexpr const char* ipAddrStr = "10.2.1.1";
-    static constexpr uint16_t radioModuleSourcePort = 12000;
-    static constexpr uint16_t radioModuleCommandSourcePort = 12001;
+    const char* ipAddrStr = (CREATE_IP_ADDR(NNetworkDefs::RADIO_MODULE_IP_ADDR_BASE, 1, CONFIG_MODULE_ID)).c_str();
```

Comment:
can you test this with ASAN? i think this is a dangling pointer (but also can just say closed wont fix until #239

Reply:
It is LOL. Dw itll be fixed eventually

Reply:
Let me shit out more features before fixing a dangling pointer

Reply:
Need all the file stuff done first